src/api/check-config.js

```diff
@@ -17,10 +17,7 @@ function getOptions({ supportedTemplates }) {
     },
     template: {
       validate(input) {
-        return (
-          supportedTemplates.includes(input) ||
```

Comment:
This check was unnecessary because the `input` is always a relative path now (not a string like `"InstantSearch.js"`).